scripts/Dockerfile.devel

```diff
@@ -29,6 +29,8 @@ RUN dnf -y install \
 	librados-devel \
 	librbd-devel \
     && dnf -y update \
+    && dnf clean all \
+    && rm -rf /var/cache/yum \
```

Comment:
This image is not pushed to any (public) registry. Only locally on developers machines and in the CI. This is fine if the savings are important.

Reply:
We don't need it, but I have not experienced any issues by having it either. As this is a final image for developers, cleaning the packages is fine IMHO. Not sure if deleting `/var/cache/yum` causes issues in case someone wants to manually run the container and install some other tools for extra debugging?

Reply:
The extra debugging in container land mostly by attaching the debugging container and doing stuffs, so I dont think thats has to be considered here.